entrypoint.sh

```diff
@@ -25,5 +27,12 @@ if [ -z "$tag" ]; then
   params=$(eval echo --progress plain -f "$file" "$BUILDPARAMS" "$path")
 fi
 
+if [ "$global" = "true" ]; then
```

Comment:
I don't know if it's supported on sh to use = I think you have to use -eq operator

Reply:
`=` is ok for strings.

From https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html

> string1 = string2
> True if the strings are equal. ...
> 
> arg1 OP arg2
> OP is one of ‘-eq’, ‘-ne’, ‘-lt’, ‘-le’, ‘-gt’, or ‘-ge’. ... Arg1 and arg2 may be positive or negative integers. ...